compiler/ballerina-lang/src/main/java/io/ballerina/projects/PackageResolution.java

```diff
@@ -476,6 +491,9 @@ private void addPackageToGraph(ResolutionResponse resolutionResponse) {
         }
 
         private List<PackageName> getPossiblePackageNames(ImportModuleRequest importModuleRequest) {
+            if (langLibPackage(importModuleRequest.packageOrg(), importModuleRequest.moduleName())) {
```

Comment:
Can you please explain why we need to special-case langlib modules here?

Reply:
I came across an issue where only `lang` was returned as possible package names. Looks like it was due an issue in a different place. I tested this now by removing it and it works.

Reply:
Fixed with b139f6d